operator/controllers/execution/scans/parse_reconciler.go

```diff
@@ -91,6 +91,21 @@ func (r *ScanReconciler) startParser(scan *executionv1.Scan) error {
 	var backOffLimit int32 = 3
 	truePointer := true
 	falsePointer := false
+
+	resources := corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("200m"),
+			corev1.ResourceMemory: resource.MustParse("100Mi"),
+		},
+		Limits: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("400m"),
+			corev1.ResourceMemory: resource.MustParse("200Mi"),
+		},
+	}
```

Comment:
Wouldnt it be easier to use default values in the CRD definition instead of hardcoding it in code?
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting
This is also true for all the other hardcoded default resource limits in other files.

Reply:
There is a way to define defaults with kubebuilder: https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
yes good point that would indeed be better.

I've had problems with the kubebuilder defaults before they haven't worked consistently for me before but we can definitely give it another try. Might be a bit hard in this case thought as we don't define the individual fields ourself but pull them in from the kubernetes api types.

Reply:
Applied the defaults working nicely 👍
Unfortunately the defaults are only applied when the custom resources are applied, so just doing it with the defaults would break existing scans. So we will to use both methods until v4 to make sure existing installations with upgraded operators keep working.